spec/license_spec.rb

```diff
@@ -27,6 +27,22 @@
           expect(approved_licenses).to include(id), "See https://git.io/vzCTV."
         end
       end
+
+      context "minimum permissions" do
+        permissions = license["permitted"]
```

Comment:
I'll be honest, I don't fully get it, but stylistically, for tests, this should be `let(:permissions) { license["permitted"] }`, similar to the `id` line, a few lines above. It's a Ruby/Rspec thing and has something to do with scopes or cleanup, IIRC.